WireUI/Sources/WireIndividualToTeamMigrationUI/Views/ConfirmationView.swift

```diff
@@ -0,0 +1,79 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDesign
+
+struct ConfirmationView: View {
+
+    enum Action {
+        case goBack
+        case `continue`
+    }
+
+    @State var password: String = ""
+    @State var migrationConfirmed: Bool = false
+    @State var termsAccepted: Bool = false
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 24) {
+            Text(String.localized(key: "individualToTeam.confirmation.body", bundle: .module))
```

Comment:
question: do we have to specify `.module` here or is the default looking at the localizable table of the module?

Reply:
Because the documentation for `String.init(localized:table:bundle:locale:comment:)` states that if the `bundle` parameter is `nil` (and it defaults to `nil`), `an app searches its main bundle.`, which obviously doesn't contain our module's strings.

But our method could default to `.module`. It really depends on how you intend to use it.

Reply:
What do you want to do @netbe ?

Reply:
I am not sure to follow, it seems we don't have a choice specify `.module` in the method. This is fine for now, I guess at some point we can use the swiftgen constants or have a convenience method for it

Reply:
The bundle argument is required.
However, I would have just written:
```suggestion
            Text("individualToTeam.confirmation.body", bundle: .module)
```